README.md

```diff
@@ -1,84 +1,80 @@
 # Celo Rosetta
 
+[![CircleCI](https://circleci.com/gh/celo-org/rosetta/tree/master.svg?style=shield)](https://circleci.com/gh/celo-org/rosetta/tree/master)
+[![License](https://img.shields.io/github/license/celo-org/rosetta.svg)](https://github.com/celo-org/rosetta/blob/master/LICENSE.txt)
+
 A monitoring server for celo-blockchain
 
-## Overview
+## What is Celo-Rosetta?
 
+Celo Rosetta is an rpc server that exposes an API to:
+ * Query Blockchain
+ * Obtain Balance Changing Operations
+ * Construct Airgapped transactions
 
-## Starting rosetta
+ 
+With a special focus on getting balanche change operations, provides an easy way to obtain changes that are not easily queryable using
+the celo-blockchain rpc; such as:
 
-To run rosetta do:
-```
-    rosetta run [options]
-```
+* Gas Fee distribution
+* Gold transfers (internal & external). Taking in account Tobin Tax
+* Epoch Rewards Distribution
+* LockedGold & Election Operations
 
-Where rosetta is the binary. 
+## RPC endpoints
 
-  * If on development you can replace rosetta by `go run main.go`
+Rosetta exposes the following endpoints:
```

Comment:
Is Rosetta == Celo Rosetta?

Reply:
Yeah... agree, we should ask Patrick what the terminology should be for the implementations, eg. `Celo Rosetta Interface`?

Reply:
he said rosetta it's fine, since it's within `celo-org` namespace.

I'll be using 'celo rosetta' in the readme